File: src/0001-two-sum/solution.py
# coding:utf-8


class Solution:

    def twoSum(self, nums, target):
        result = []
        count = len(nums)
        for i in range(count - 1):
            for j in range(i + 1, count):
                if nums[i] + nums[j] == target:
                    result = [i, j]
                    break
        return result


solution = Solution()
nums = [2, 4, 3]
target = 6
ret = solution.twoSum(nums, target)
print(ret)

File: src/0001-two-sum/solution.php
<?php

class Solution
{
    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    public function twoSum($nums, $target)
    {
        $result = [];
        $count = count($nums);
        for ($i = 0; $i < $count - 1; $i++) {
            for ($j = $i + 1; $j < $count; $j++) {
                if ($nums[$i] + $nums[$j] == $target) {
                    $result = [
                        $i, $j,
                    ];
                    break;
                }
            }
        }
        return $result;

    }
}

$nums = [3, 2, 4];
$target = 6;
$solution = new Solution;
$result = $solution->twoSum($nums, $target);
var_dump($result);
